src/GeositeFramework/sample_plugins/identify_point/main.js

```diff
@@ -22,6 +22,11 @@ define(["use!Geosite", "dojo/_base/declare", "framework/PluginBase"],
 
             initialize: function(args) {
                 declare.safeMixin(this, args);
+
+                if (N.app.singlePluginMode) {
```

Comment:
I don't think we've previously advertised the internal api to the apps.  Perhaps this setting should be transferred to the `app` object that comes in `initialize(args)` so we can maintain some independence with the internal implementation.

Reply:
Good call. Pushed some fixup commits.